### DISCUSSION TITLE
is the config working for you?

### DISCUSSION THREAD

**Discussion by mitvitaminen** [projectM-visualizer/projectm#646] [Polls]
Hi guys,

I have an issue in seeing a difference when changing settings no matter which I twiddle with projectM looks the same. Is the config I can set when ctrl-m open the settings there? They are getting persisted because when I open after closing the app the values do stay as I wrote. I mean should this not react on beat like crazy either with beat sensitivity real low or real high one should work right if I misinterpret the help text. I think the only thing that changes to settings is the starting window size and checkboxes not checked i admit. This is on arch linux btw.

kind regards do not forget to  vote,
Chris

**Comment by kblaschke** [comment]
I think you're talking about the old `config.inp` file. This had numerous issues and is basically gone in the current development code.

You can try one of my development releases here which matches your OS, it should work better than the old app and config:
https://github.com/kblaschke/frontend-sdl2/releases
